src/devices/bus/nes/bootleg.cpp

```diff
@@ -2148,7 +2148,7 @@ void nes_yung08_device::write_45(offs_t offset, u8 data)
 	switch (offset & 0x51ff)
 	{
 		case 0x4022:
-			prg8_cd(data & 1 ? 3 : 4 + ((data & 0x07) >> 1));
+			prg8_cd(data & 1 ? 3 : 4 + BIT(data, 1, 2));
```

Comment:
Doesn’t this change behaviour?  Previously it was selecting three bits, now it’s selecting two.

Reply:
((data & 7) >> 1) == (data >> 1 & 3)

Reply:
Oh, it was doing the mask first and shifting an included bit off the end. Nasty.

Reply:
The Taito board in this PR was likewise masking 7 bits, shifting and keeping 6. There are certainly more examples.